.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,40 @@
 ## Problem
 
-Test
+<!-- Describe the issue this PR resolves. -->
+<!-- Add a link to the github issue with its number (e.g. #123) -->
+
+## Description
+
+<!-- Provide a top-level description of the changes in this PR. -->
+<!-- or a link to the original issue that contains this information -->
+<!-- Ensure the reviewer is able to understand, in general terms, what you're changing and why. -->
+
+## Pre-Requisites
+
+<!-- Anything other than access to, and a good working knowledge of, the environment needed to test this PR. -->
+<!-- For example, a HANA database with a specific tenant database, or an NFS server having specific shares -->
+<!-- Dependency versions also belong here. -->
+<!-- If you have prepared resources in advance to make the reviewer's life easier, record the information here. -->
+<!-- Explicitly state "None", if there are none. -->
+
+## Commitment to Test
+
+- Testing takes ?<!-- Roughly how long will it take to complete the testing? -->
+- Reviewing ? files <!-- Roughly how many files will need review - fewer than 5, 10, 20 ... ? -->
+<!-- In short, anything you consider relevant which answers the question in the reviewer's head: "Do I think I can review this right now?" -->
+
+## Test Instructions
+
+<!-- List the instructions the reviewer will need to follow to ensure this works correctly. -->
+<!-- Try to make it easy for the reviewer to test this PR. -->
+
+## Acceptance Criteria
+
+<!-- How can the reviewer know the aims of the PR have been achieved? -->
+<!-- Ensure any post-test actions needed for confirmation are included here. -->
```

Comment:
I'm increasingly of the opinion these should be merged into

    ## Test Instructions / Acceptance Criteria

Reply:
Often they end up being the same, but really they are different things.

How you should test, versus the things that make this good enough as a solution